cs/troubleshooting.texy

```diff
@@ -72,6 +72,20 @@ Rozšíření Apache nazvané mod_rewrite je potřeba povolit, aby bylo možné
 	RewriteRule !\.(pdf|js|ico|gif|jpg|png|css|rar|zip|tar\.gz)$ index.php [L]
 \--
 
+.[note]
+Pro ovlivňování chování Apache soubory `.htaccess` je třeba mít povolenou direktivu `AllowOverride`. Toto je v Apache výchozí chování.
+
+Pro nginx je třeba nastavit přesměrování pomocí direktivy `try_files` uvnitř bloku `location /` v konfiguraci serveru.
+
+/--
+	location / {
+		try_files $uri $uri/ /index.php$is_args$args;
+	}
+\--
+
+.[note]
+Block `location` se pro každou filesystémovou cestu smí v bloku `server` vyskytovat jen jednou. Pokud již v konfiguraci `location /` máte, přidejte direktivu `try_files` do něj.
```

Comment:
better but not ideal is
Blok `location` se pro každý adresář smí v bloku `server` vyskytovat pouze jednou.